infra/gcp/bash/ensure-prod-storage.sh

```diff
@@ -87,7 +89,9 @@ readonly PROD_PROJECT_DISABLED_SERVICES=(
 )
 
 # Regions for prod GCR.
-PROD_REGIONS=(us eu asia)
+GCR_PROD_REGIONS=(us eu asia)
+# Regions for prod AR.
+AR_PROD_REGIONS=(asia-east1 asia-south1 asia-northeast1 australia-southeast1 europe-north1 europe-southeast1 europe-west1 europe-west4 southamerica-west1 us-central1 us-east1 us-east4 us-west2)
```

Comment:
@upodroid can we start with the locations of Cloud Run subject to Tier 1 pricing https://cloud.google.com/run/docs/locations ?

Reply:
Added all Tier 1 regions. 

Will bring this up at next week's sig-release meeting